tests/test_visualize.py

```diff
@@ -272,7 +257,7 @@ def test_lift_curve_have_correct_data(self, classifier):
         assert "Lift" == ax.get_ylabel()
         assert "% of Data" == ax.get_xlabel()
         assert pytest.approx(19.5) == np.sum(ax.lines[0].get_xdata())
-        assert pytest.approx(49.849, rel=0.0001) == np.sum(ax.lines[0].get_ydata())
+        assert pytest.approx(60.291, rel=0.0001) == np.sum(ax.lines[0].get_ydata())
```

Comment:
Shuffled data?

Reply:
Yeah, the dataset gets moved around a bit, so the randomness occurs in different order